tools/validate_reflection.py

```diff
@@ -419,12 +442,12 @@ def read(self):
                     debug("  %s" % g)
                 debug("}")
                 assert len(match_2_comments.groups()) == 7, "match_2_comments wrong size due to regex pattern change"
-                ignore_or_swap1 = match_2_comments[2]
-                next_reflect_ignore_swap1 = match_2_comments[3]
-                ignore_or_swap2 = match_2_comments[4]
-                next_reflect_ignore_swap2 = match_2_comments[5]
-                search_string_for_next_reflect_class = match_2_comments[6]
-                next_reflect_class = match_2_comments[7]
+                ignore_or_swap1 = match_2_comments.group(2)
```

Comment:
This series of assignments could be written as:
```python
(ignore_or_swap1, 
next_reflect_ignore_swap1,
ignore_or_swap2,
next_reflect_ignore_swap2,
search_string_for_next_reflect_class,
next_reflect_class) = match_2_comments.group(*range(1, 8))
```
to eliminate the right hand side repetition.  (If the \_re_Match_ object behaves the way re.Match does.)